pkg/trace/writer/config/trace_writer.go

```diff
@@ -6,6 +6,7 @@ import "time"
 type TraceWriterConfig struct {
 	FlushPeriod      time.Duration
 	UpdateInfoPeriod time.Duration
+	MaxConnections   int
```

Comment:
We have this here and another one in `SenderConfig.MaxConnections`.

Reply:
Arghh... Some weird problem with VIM lately. It's not reloading files. I have no idea what I broke... Fixed!